features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -155,10 +155,10 @@ export default function Configuration() {
                 return nextState;
             case 'accessControlRoles':
                 // TODO: need to do the role validation here ~tmkb
-                return { ...copyAPIConfig(state), [action]: value.split(',') };
+                return { ...copyAPIConfig(state), [action]: value ? value.split(',') : '' };
```

Comment:
Shall we change the empty sting to an empty array because REST API accept `accessControlRoles` field as an array
```javascript
                return { ...copyAPIConfig(state), [action]: value ? value.split(',') : [] };
```

Reply:
Fixed in f20d89587c00c4aadaf582e15f4f469743872571